spec/std/openssl/ssl/server_spec.cr

```diff
@@ -76,4 +76,21 @@ describe OpenSSL::SSL::Server do
       end
     end
   end
+
+  it "detects SNI hostname" do
+    tcp_server = TCPServer.new(0)
+    server_context, client_context = ssl_context_pair
+
+    OpenSSL::SSL::Server.open tcp_server, server_context do |server|
+      spawn do
+        sleep 1
+        OpenSSL::SSL::Socket::Client.open(TCPSocket.new(tcp_server.local_address.address, tcp_server.local_address.port), client_context, hostname: "example.com") do |socket|
```

Comment:
Could you please split this line? Its complexity makes it hard to read.

Reply:
I can, keep in mind its a copy paste of the above tests, I just wanted to keep the same structure, maybe it should be better to merge this PR and make a new one fixing all the examples in the file which use this exact same line?